src/main/java/com/zs/assignment1/SystemInfo.java

```diff
@@ -0,0 +1,23 @@
+package com.zs.assignment1;
+
+import java.io.File;
+
+public class SystemInfo {
+    public static void main(String[] args){
+        String username= System.getProperty("user.name");
```

Comment:
Not required to create variable for this. As you are just printing it and not using it anywhere else. You could have just evaluate the value and print n the println() method only.